includes/Channel.hpp

```diff
@@ -74,7 +74,7 @@ class Channel {
   const Mode &getMode(void) const;
   const std::string getChannelModeFlag(void) const;
   const std::string &getKey(void) const;
-  uint8_t getLimit(void) const;
+  int getLimit(void) const;
```

Comment:
Same as above:
I wonder if it's better to use unsigned type if we won't have negative values. (It'll depend on how you use it in the code.)